### PR TITLE
Add donation progress to sidebar; WIP

### LINKED ISSUE #679: Show donation-progress component in the project sidebar
# Problem

We implemented the donation-progress component, but it hasn't been placed above the "Donate" button in the project's sidebar.

See: https://projects.invisionapp.com/share/SP8PSQGNM#/screens/191332624

## Subtasks
- [ ] Add the donation-progress component into the sidebar
- [ ] Test that it's there

### PR DESCRIPTION
**Pull request by JoshSmith**
# What's in this PR?

Minor progress on adding the donation progress bar to the sidebar. Not fully hooked up and really need to figure out Mirage.

## References
Fixes #679

### CODE REVIEW

**Comment by JoshSmith** [comment]
@alexwilkinson I implemented some of this. Feel free to take over from here if you'd like.

**Comment by alexwilkinson** [comment]
Awesome! Since I'm still at my conference, feel free to complete this if you'd like. I only really managed to get the apps set up before I left. 

On Sat, Nov 19, 2016 at 2:15 AM Josh Smith

<
mailto:Josh Smith <notifications@github.com>
> wrote:

a, pre, code, a:link, body { word-wrap: break-word !important; }

https://github.com/alexwilkinson
I implemented some of this. Feel free to take over from here if you'd like.

—

You are receiving this because you were mentioned.

Reply to this email directly,
https://github.com/code-corps/code-corps-ember/pull/694#issuecomment-261700704
, or

**Comment by JoshSmith** [comment]
Oh wow, that's a long conference! I may not get to it since I'm switching over to some API stuff, so if it's still open when you're back around then let me know.

File: app/components/organization-profile.js [hunk 1]
```diff
@@ -11,8 +10,6 @@ export default Component.extend({
 
   credentials: service(),
 
-  organizationMembers: mapBy('organization.organizationMemberships', 'member'),
```
Comment (JoshSmith) [inline app/components/organization-profile.js]: Not sure why I removed this.

File: app/controllers/project/settings/donations.js [hunk 2]
```diff
@@ -76,8 +83,15 @@ export default Controller.extend({
      */
     saveDonationGoal(donationGoal, properties) {
       donationGoal.setProperties(properties);
+      console.log('set');
```
Comment (JoshSmith) [inline app/controllers/project/settings/donations.js]: When the donation goal `hasDirtyAttributes` it gets sorted to the top of the list for some reason. I tried a number of things to fix this but no dice. In particular, I thought the transform would affect it. This seemed to make no difference.

Comment (begedin) [inline app/controllers/project/settings/donations.js]: Maybe we ought to explicitly sort the list in some way, then. By ID, maybe? Last updated? Amount?

**Pull request merged**